docs/containers.md

```diff
@@ -85,3 +88,92 @@ ERROR at annotation (semantic) stage
 pyccel:
  |fatal [semantic]: foo.py [4,10]| Inhomogeneous tuples must be indexed with constant integers for the type inference to work (a)
 ```
+
+## Lists/Sets/Dictionaries
+
+Homogeneous lists, sets and dictionaries are implemented using external libraries. In C we rely on [STC](https://github.com/stclib/STC). In Fortran we rely on [gFTL](https://github.com/goddard-Fortran-Ecosystem/gFTL/).
+
+For example the following code:
+```python
+def f():
+    my_list = [1, 2, 3, 4]
+    my_set = {1, 2, 3, 4}
+    my_dict = {1:1.0, 2:2.0}
+    b = my_list[0]+2
+    return b + my_set.pop() + my_dict[1]
+```
+is translated to the following C code:
+```c
+double f(void)
+{
+    vec_int64_t my_list = {0};
+    hset_int64_t my_set = {0};
+    hmap_int64_t_double my_dict = {0};
+    int64_t b;
+    double result;
+    my_list = c_init(vec_int64_t, {INT64_C(1),INT64_C(2),INT64_C(3),INT64_C(4)});
+    my_set = c_init(hset_int64_t, {INT64_C(1),INT64_C(2),INT64_C(3),INT64_C(4)});
+    my_dict = c_init(hmap_int64_t_double, {{INT64_C(1), 1.0}, {INT64_C(2), 2.0}});
```

Comment:
Should we not use `c_make` here?

Reply:
Oops. Generated before I updated the branch.
Fixed: [cda5627](https://github.com/pyccel/pyccel/pull/2270/commits/cda562726bc1925bc5b2cd43f29587747e7dd3fd)